packages/terra-embedded-content-consumer/docs/TestingEmbeddedApplications.md

```diff
@@ -0,0 +1,7 @@
+# Testing Embedded Content
+
+TThe Embedded Content Provider Test Template is a test template component which should be used to verify that the the provided content does indeed resize and behavior as expected when embedded in an terra-embedded-content-consumer XFC Consumer iframe. 
```

Comment:
nit: There is a typo, `TThe`, at the beginning of the sentence. It also looks the like word `the` is duplicated in the sentence, `verify that the the provided content`.
Need to change `does indeed resize and behavior as expected` to `does indeed resize and behave as expected`